GA/post.py

```
@@ -0,0 +1,1126 @@
+        - fname : Path
+            The filename or path of the .dill file to be loaded.
+            If not an absolute path it will be relative to base_dir.
+        - base_dir : Path, optional
+            The base directory to use if fname is not an absolute path.
+            Defaults to the directory containing this script. 
+
+        Returns
+        -------
+        None
+        """
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)